build.sbt

```diff
@@ -10,37 +11,38 @@ majorVersion              := 1
 retrieveManaged           := true
 
 lazy val lisa = Project(appName, file("."))
-enablePlugins(PlayScala, SbtAutoBuildPlugin, SbtGitVersioning, SbtDistributablesPlugin)
+enablePlugins(PlayScala, SbtDistributablesPlugin)
 
 scalaSettings
 publishingSettings
 defaultSettings()
-integrationTestSettings()
-configs(IntegrationTest)
 
 scalaVersion := "2.12.12"
 
-val testScope = "test"
+val silencerVersion = "1.7.1"
 
 libraryDependencies ++= Seq(
   ws,
-  "uk.gov.hmrc"             %% "bootstrap-backend-play-27" % "4.2.0",
-  "uk.gov.hmrc"             %% "auth-client"               % "5.2.0-play-27",
+  "uk.gov.hmrc"             %% "bootstrap-backend-play-27" % "5.3.0",
   "uk.gov.hmrc"             %% "domain"                    % "5.11.0-play-27",
-  "org.scalatest"           %% "scalatest"                 % "3.0.8"             % testScope,
-  "org.pegdown"             %  "pegdown"                   % "1.6.0"             % testScope,
-  "org.mockito"             %  "mockito-core"              % "3.3.0"             % testScope,
-  "com.typesafe.play"       %% "play-test"                 % PlayVersion.current % testScope,
-  "org.scalatestplus.play"  %% "scalatestplus-play"        % "3.1.3"             % testScope
+  "org.scalatest"           %% "scalatest"                 % "3.0.9"             % Test,
+  "org.pegdown"             %  "pegdown"                   % "1.6.0"             % Test,
+  "org.mockito"             %  "mockito-core"              % "3.10.0"            % Test,
+  "com.typesafe.play"       %% "play-test"                 % PlayVersion.current % Test,
+  "org.scalatestplus.play"  %% "scalatestplus-play"        % "4.0.3"             % Test,
+  compilerPlugin("com.github.ghik" % "silencer-plugin" % silencerVersion cross CrossVersion.full),
+  "com.github.ghik" % "silencer-lib" % silencerVersion % Provided cross CrossVersion.full
 )
-unmanagedResourceDirectories in Compile += baseDirectory.value / "resources"
 
-evictionWarningOptions in update        := EvictionWarningOptions.default.withWarnScalaVersionEviction(false)
+Compile / unmanagedResourceDirectories += baseDirectory.value / "resources"
+
+Test / fork                            := true
+Test / parallelExecution               := false
```

Comment:
redundant, parallelExecution is already set to false in defaultSettings: https://github.com/hmrc/sbt-settings/blob/master/src/main/scala/uk/gov/hmrc/DefaultBuildSettings.scala#L64